source/documentation/standards/naming-domains.html.md.erb

```diff
@@ -81,39 +85,50 @@ For example: `staff.staging.prisonvisits.service.gov.uk`.
 This pattern makes the relationships between domains clear, grouping
 environments by service and applications by shared environment.
 
-Pre-production environments and [hosted prototypes](https://www.gov.uk/service-manual/design/making-prototypes#sharing-code-prototypes)
-should also use authentication (such as HTTP basic auth) to prevent
-public users who come across them thinking they’re real.
+Non-production environments and [hosted prototypes]
+(https://www.gov.uk/service-manual/design/making-prototypes#sharing-code-prototypes)
+should also use authentication (such as HTTP basic auth) to prevent public
+users who come across them thinking they’re real.
+
+Non-production environments can also use different styling as a way of making it
+clear to users which environment they’re in - but that’s more useful for people
+working on the service to not modify data in production than to keep users out.
+GOV.UK publishing apps do this, for example compare GOV.UK Signon in their
+[Staging](https://signon.staging.publishing.service.gov.uk) and [Production]
+(https://signon.publishing.service.gov.uk) environments.
+
+## Non `.gov.uk` domains
```

Comment:
What about the .org.uk type domain that have to be approved by exception by GDS? Should we mention that or link to that process here?

Reply:
I've [added an explainer](https://github.com/ministryofjustice/technical-guidance/pull/124/files#diff-9724b49a8c72933f3acd98846b1dbdc4fed08d5dec426b948d86e6e77da20095R137) pointing toward the [GOV.UK Proposition](https://www.gov.uk/government/publications/govuk-proposition/govuk-proposition), how to get an exemption, and what to do when you have one.

Happy to add more to it if you'd like? @AntonyBishop

Reply:
LGTM